### PR TITLE
Preset thumbnails 2

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Finalized preset thumbnails, and did some fixes with the communication with BeNine

### CODE REVIEW

File: objects/BenineHelper.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Preset from "./Preset";
 
 // Singleton Object
 let benineHelperInstance = null;
-const serverAddress = "localhost";
+const serverAddress = "192.168.0.12";
```
Comment (Markvanderuit) [inline objects/BenineHelper.js:7]: Let's not forget about this tomorrow. Might prove troubling.

**Comment by Markvanderuit** [comment]
Nothing else to comment, LGTM.

File: objects/BenineHelper.js (hunk 1 above)
Comment (bart-de-jonge) [inline objects/BenineHelper.js:7]: I won't this is the only setting I will need to change tomorrow

**Comment by geenen124** [comment]
Looking forward to playing with this tomorrow 😄

**Pull request merged**